wasm/store_test.go

```diff
@@ -52,3 +54,45 @@ func TestBuildFunctionInstances_FunctionNames(t *testing.T) {
 	// We expect unknown for any functions missing data in the NameSection
 	require.Equal(t, []string{"unknown", "two", "unknown", "four", "five"}, names)
 }
+
+func TestValidateAddrRange(t *testing.T) {
+	name := "test"
+	s := NewStore(nil)
+	mi := s.getModuleInstance(name)
+
+	m := &Module{
+		MemorySection: []*LimitsType{{Min: 100}},
+	}
+
+	_, err := s.buildMemoryInstances(m, mi)
+	require.NoError(t, err)
+
+	require.True(t, mi.Memory.ValidateAddrRange(uint32(0), uint64(0)))
```

Comment:
same comment here about matrix to document why the inputs or results are interesting

Reply:
Improved. Thanks!

Reply:
I surely fixed it this time. Thanks.